### ISSUE TITLE
Android Lollipop atomic support is not avaiable for ppc64el, s390x

### ISSUE THREAD

**Issue by vicamo** [laarid/package_android-system-core#4]
This can be resolved by uplifting code base to marshmallow or above, which use __atomic_int_least32_t__ from compiler support of [c++11](http://en.cppreference.com/w/c/atomic).

**Comment by vicamo** [comment]
Tag also armel, which causes error:
```
Error: selected processor does not support `dmb' in ARM mode
```
It's really the same thing but just the preprocessor doesn't take armel into consideration.

**Comment by vicamo** [comment]
Closed by #8.
